index.html

```diff
@@ -1186,6 +1186,23 @@ <h1>The Registry</h1>
           <a href="https://github.com/hashgraph/did-method/blob/master/did-method-specification.md">Hedera Hashgraph DID Method</a>
       </td>
   </tr>
+  <tr>
+    <td>
+        did:sirius:
+    </td>
+    <td>
+        PROVISIONAL
+    </td>
+    <td>
+        ProximaX Sirius Chain
+    </td>
+    <td>
+        ProximaX Enterprise, Proximax Inc.
+    </td>
+    <td>
+        <a href="https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md">ProximaX SiriusID DID Method</a>
```

Comment:
This document does not appear to define Create, Read Update Deactivate: https://w3c-ccg.github.io/did-method-key/#operations

Or Privacy / Security considerations...for example:

https://w3c-ccg.github.io/did-method-key/#security-and-privacy-considerations

I'd prefer to see more explicit definitions.

Reply:
Added more details on the DID method specs. We will add more details as we see fit.

https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md

@OR13